index.html

```diff
@@ -20,19 +20,19 @@
     <a href="logo" class="logo">My logo</a>
     <nav>
       <div class="mobile-navbar" id="mobile-navbar">
+        <img class="exit-btn" width="18" height="16" src="./images/cancel-icon.png" alt="hamburger cancel button">
         <ul class="mobile-nav-menu" id="mobile-nav-menu">
-          <li class="mobile-nav-link"><a href="#works">Portfolio</a></li>
-          <li class="mobile-nav-link"><a href="#about-myself">About</a></li>
-          <li class="mobile-nav-link"><a href="#form-section">Contact</a></li>
+          <li class="mobile-nav-link"><a href="#works-sect">Portfolio</a></li>
+          <li class="mobile-nav-link"><a href="#about-me">About</a></li>
+          <li class="mobile-nav-link"><a href="#form">Contact</a></li>
         </ul>
```

Comment:
- [ ] It is better to wrap the x button and the menu items in a div and give it background: rgb(0, 0, 255) and opacity: 0.6 to make the mobile menu looks like the Figma design.

Your Design | Figma Design
---------------- | --------------------
![Screenshot from 2022-06-21 16-15-41](https://user-images.githubusercontent.com/33752735/174821817-4c5e0cc7-05a4-46c5-a7f9-b55f0c952eaa.png) | ![Screenshot from 2022-06-21 16-09-48](https://user-images.githubusercontent.com/33752735/174821850-289b6411-3bb7-4c8f-9793-c8635f013299.png)